CMakeLists.txt

```diff
@@ -63,3 +63,4 @@ ign_create_docs(
   TUTORIALS_MAINPAGE_MD "${CMAKE_BINARY_DIR}/tutorials.md"
   )
 
+add_subdirectory(cmd)
```

Comment:
The rest of the CLI stuff is in `loader/conf` - how about putting the autocomplete stuff there too? See #32 as a reference.

Reply:
https://github.com/gazebosim/gz-plugin/commit/3f84528404f8965b3c7d552734134318ea77096d + https://github.com/gazebosim/gz-plugin/commit/b078cd1cc7ca6b61c141c20fa12f74281321a910